custom_components/nordpool/sensor.py

```diff
@@ -349,6 +357,24 @@ def _update(self):
         self._peak = mean(today[8:20])
         self._mean = median(today)
 
+        if self.tomorrow_valid:
+            tomorrow = self.tomorrow
+            self._tomorrow_average = mean(tomorrow)
+            self._tomorrow_min = min(tomorrow)
+            self._tomorrow_max = max(tomorrow)
+            self._tomorrow_off_peak_1 = mean(tomorrow[0:8])
```

Comment:
In every country off peak can be at different time.

Reply:
We use what nordpool defines as off peak but we account for local time.